src/patternfly/components/FormControls/docs/code.md

```diff
@@ -8,3 +8,6 @@ Input, Textarea, and select are provided in the form controls component for use
 | Class | Applied To | Outcome |
 | -- | -- | -- |
 | `.pf-c-form-control` | `<input>`,`<textarea>`, `<select>` |  Initiates an input, textarea or select. For styling of checkboxes or Radios see the <a href="/components/Check/examples/">check component</a>. **required**  |
+| `.pf-m-form-control-alt` | `input[type="text"].pf-c-form-control` |  Applies alternative form control styles. |
```

Comment:
I completely forgot what `.pf-m-ghost` is for, and am wondering if that's what we should use here? I doubt we'd ever have more than one `alt` form control, but just wondering if we have an existing modifier that's more descriptive and applicable for this case. But maybe for theming, `alt` does make the most sense.

Also, if we stick with `alt`, should we drop the `form-control` and just have `.pf-m-alt`?

Reply:
ya I don't really like alt but to be honest I couldn't think of anything else. I'm not sure about having a pf-m-alt as a general modifier it feels like that could be misused. I have no idea what else to call this though. I did have it as pf-m-form-control-login I think but @andresgalante didn't like it. 

Ghost is a bit different right, it's the absence of borders altogether. It could work here though. I am open to any other suggestions :)

Reply:
@matthewcarleton is there a reason we have multiple types of inputs?  This is not a pattern I have seen from other CSS Frameworks. If we truly need a different style for login forms (seems odd to me), it seems that could be achieved by modifying the CSS variables in the login form.

Reply:
Hey @dmiller9911 this is a great question for design.I believe @mceledonia wanted to distinguish this type of form from the others. We talked about if this should be a modifier on the input or the form itself. It made sense to keep it as a modifier on the input b/c we are styling the inputs as a component. I'm happy to discuss this further but I think it makes sense to do it outside this PR just so it doesn't hold this one up.

Reply:
@jgiardino there is no such a thing as a `pf-m-ghost` in patternfly. We do have `m-action` and `m-link`